recipes-debian/libkrb5/libkrb5_debian.bb

```diff
@@ -0,0 +1,119 @@
+# base recipe: meta-openembedded/meta-oe/recipes-connectivity/krb5/krb5_1.17.bb
+# base branch: warrior
+# base commit: a24acf94d48d635eca668ea34598c6e5c857e3f8
+SUMMARY = "Runtime library for the Kerberos v5 API for the main part used by the application and the Kerberos client"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://NOTICE;md5=aff541e7261f1926ac6a2a9a7bbab839"
+
+DEPENDS = "bison-native ncurses util-linux e2fsprogs e2fsprogs-native openssl"
+
+inherit autotools-brokensep binconfig debian-package
+require recipes-debian/sources/krb5.inc
+
+DEBIAN_UNPACK_DIR = "${WORKDIR}/krb5-${PV}"
+
+SRC_URI += "file://0001-aclocal-Add-parameter-to-disable-keyutils-detection.patch;patchdir=src \
```

Comment:
Where did you get those patch from? and Why need them?
Please describe these reason in commit log or recipe.